src/WebJobs.Script/Description/DotNet/FunctionAssemblyLoadContext.cs

```diff
@@ -139,11 +159,16 @@ private bool TryLoadRuntimeAssembly(AssemblyName assemblyName, out Assembly asse
             assembly = null;
             try
             {
+                _sharedContextAssembliesInFallbackLoad.TryAdd(assemblyName.Name, null);
```

Comment:
Is this to track assemblies in the act of being loaded?

Reply:
Essentially, yes.

Reply:
More specifically, when they're being loaded by the shared context.